scripts/portfolioView.js

```diff
@@ -0,0 +1,91 @@
+(function(module){
+  var portfolioView = {};
+  var $headerPosition = $('header').offset().top;
+
+  // helper functions
+  // toggle sticky header class based on scroll position
+  function stickyHeader(){
+    if($(this).scrollTop() > $headerPosition){
+      $('header').addClass('sticky');
+    } else {
+      $('header').removeClass('sticky');
+    }
+  };
+
+  // on click build detailed view of project
+  portfolioView.detailHandler = function (){
```

Comment:
Separate into 2 functions. This performs two functions. Updating content of detail view and animating the display.